src/app/core/services/travelperk/travelperk-payment-profile-settings.service.ts

```
@@ -0,0 +1,66 @@
+  static mapAPIResponseToFormGroup(travelperkPaymentProfileSettingResponse:TravelperkPaymentProfileSettingGet[] | null): FormGroup {
+      const paymentProfileMappingsArray = travelperkPaymentProfileSettingResponse ? this.constructFormArray(travelperkPaymentProfileSettingResponse) : [] ;
+      return new FormGroup({
+          paymentProfileMappings: new FormArray(paymentProfileMappingsArray)
+      });
+  }
+
+  static createPaymentProfileSettingPayload(travelperkPaymentProfileSettingForm: FormGroup){
+      return this.constructPaymentProfileMapping(travelperkPaymentProfileSettingForm.get('paymentProfileMappings')?.value);
+  }
+
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)